emcc.py

```diff
@@ -320,7 +320,11 @@ def expand_byte_size_suffixes(value):
   value = value.strip()
   match = re.match(r'^(\d+)\s*([kmgt]?b)?$', value, re.I)
   if not match:
-    exit_with_error("invalid byte size `%s`.  Valid suffixes are: kb, mb, gb, tb" % value)
+    if value == -1:
```

Comment:
I think value is a string so this won't work.

How about you just to `if value == '-1': return -1 adter the `strip` and before the the `rs.match`.

Also we should probably add a test

Reply:
Actually, better still can you at `-?` to the beginning of the regular expression and have all negative things "just work"?

Reply:
So it would start with `r'^(-?\d+)\s...`

Reply:
@sbc100 so after adding -? and making it r'^(-?\d+)\s... we shouldn't need the if condition right ?

Reply:
Yeah, it should just work.   You should even be able to do stuff like "-2Kb" and get -2048 out :)